mantis/simulation/routers/lautaro.py

```diff
@@ -0,0 +1,201 @@
+# solves using NLP optimization (or what best underlying engine decides)
```

Comment:
Add couple notes how it solves, and what modifications made to book BF

Reply:
I will do it, but first I want to reach a satisfactory solution. I won't take more of 2/3 days